src/components/hero/_macro.njk

```diff
@@ -27,7 +27,7 @@
                                 onsButton({
                                     "type": 'button',
                                     "text": params.button.text,
-                                    "classes": 'u-mb-m',
+                                    "classes": 'btn--link u-mb-m',
                                     "url": params.button.url,
                                     "innerClasses": "icon--chevron-right-white"
```

Comment:
I think this innerClasses property can be removed here as the btn--link class does the same thing